src/store/login/login.reducer.ts

```diff
@@ -0,0 +1,245 @@
+import {
+  LoginActionTypes,
+  LoginAction,
+  WalletName,
+  AccountData,
+} from "src/store/login/login.actions";
+import { AsyncTask } from "src/utils/types";
+// domain
+import { HermezWallet } from "src/domain/hermez";
+import { AuthSignatures } from "src/domain/local-storage";
+import { getAuthSignatures } from "src/persistence/local-storage";
+
+export interface LoginState {
+  step: Step;
+  // ToDo: This state is never set. The actions do not exist.
+  // networkNameTask: AsyncTask<string, string>;
+  addAccountAuthTask: AsyncTask<null, string>;
+  accountAuthSignatures: AuthSignatures;
+}
+
+// ToDo: This should be removed when we migrate login.view.jsx module to TS
+export enum STEP_NAME {
+  WALLET_SELECTOR = "wallet-selector",
+  ACCOUNT_SELECTOR = "account-selector",
+  WALLET_LOADER = "wallet-loader",
+  CREATE_ACCOUNT_AUTH = "create-account-auth",
+}
+
+export type Step =
+  | {
+      type: "wallet-selector";
+      // ToDo: Why don't we move walletName to LoginState? It's in three steps and can be undefined...
+      walletName: WalletName | undefined;
+    }
+  | {
+      type: "account-selector";
+      // ToDo: What is this for? Isn't it the same as wallet-selector?
```

Comment:
This is remains from hardware wallet stuff

Reply:
Ok, we can remove it when we get rid of it then.